processor/processor.go

```diff
@@ -1187,26 +1230,26 @@ func (proc *Handle) getFailedEventJobs(response transformer.Response, commonMeta
 		)
 		payload, err := jsonfast.Marshal(messages)
```

Comment:
Any idea why we are storing a single job containing an array of failed payloads to proc_errors? @cisse21?
And we are using this single job to calculate dropped events as well. Wouldn't that cause skewed rsources metrics @Sidddddarth?